app/main.py

```diff
@@ -1,3 +1,57 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: float | int) -> any:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: float | int) -> any:
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, mod: float | int) -> any:
+        if isinstance(mod, (int, float)):
+            return Distance(round(self.km * mod, 3))
+
+    def __truediv__(self, other: float | int) -> any:
+        if isinstance(other, (int, float)) and other != 0:
+            return Distance(round(self.km / other, 2))
+        return NotImplemented
+
+    def __eq__(self, other: float | int) -> any:
+        if isinstance(other, Distance):
+            return self.km == other.km
+        return self.km == other
+
+    def __lt__(self, other: float | int) -> any:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        return self.km < other
+
+    def __le__(self, other: float | int) -> any:
+        if isinstance(other, Distance):
+            return self.km <= other.km
+        return self.km <= other
+
+    def __gt__(self, other: float | int) -> any:
+        if isinstance(other, Distance):
+            return self.km > other.km
+        return self.km > other
+
+    def __ge__(self, other: float | int) -> any:
+        if isinstance(other, Distance):
+            return self.km >= other.km
+        return self.km >= other
+    
```

Comment:
![image](https://github.com/user-attachments/assets/a489839a-d391-45d0-a142-b85b25c2b6a2)
![image](https://github.com/user-attachments/assets/b0d3f4b4-8229-4bd8-b99e-74e8b7e45625)

The last line isn't empty so flake returns an error. Need to remove the space